requirements.txt

```diff
@@ -9,6 +9,7 @@ scikit-learn
 pandas
 json_tricks
 h5py
+pymatreader
```

Comment:
should not be necessary

https://github.com/mne-tools/mne-python/blob/main/requirements_hdf5.txt

Reply:
Oh, good catch

Reply:
```suggestion
```